package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/metamorphoses",
-  "version": "0.13.3",
+  "version": "0.13.4",
```

Comment:
We can do it later. I want to run performance tests instead of waiting for those changes to land.